PyTDF/CallableGenerator.py

```diff
@@ -26,7 +27,7 @@ def _dfs(self, node, prev = 't', ops = []):
                 self._dfs(n)
             return
 
-        if node.operation.op_type == "a":
+        if node.operation.op_type == node.operation.Types.ACTION:
```

Comment:
As I already pointed out,`Types.ACTION` needs to be accessed as a class data member: `Operation.Types.ACTION`, not as an instance data member as you do here. Please make sure you address our comments, and ask if we are not clear enough!

Reply:
@etejedor , thanks for pointing out. Sorry that I didn't realize this. I'll fix this :+1: